nsot/models.py

```diff
@@ -1547,6 +1548,19 @@ class Circuit(Resource):
         )
     )
 
+    # This doesn't use the built-in SlugField since we're doing our own
+    # slugification (django.utils.text.slugify() is too agressive)
+    name_slug = models.CharField(
+        db_index=True,
+        editable=False,
+        max_length=255,
+        null=True,
+        unique=True,
```

Comment:
I tried `default=''`, but that breaks the `unique` constraint which causes the `0030_add_circuit_name_slug` to fail. I felt like keeping the uniqueness constraint to allow indexing was more important that providing a default value.